data/top_config.toml

```diff
@@ -85,6 +85,15 @@ block_ios = [{ block = "uart", instance = 2, io = "tx" }]
 name = "rs232_rx"
 block_ios = [{ block = "uart", instance = 2, io = "rx" }]
 
+[[pins]]
+name = "rs485_rx"
+block_ios = [{ block = "uart", instance = 2, io = "rx" }]
+
+[[pins]]
+name = "rs485_tx"
+block_ios = [{ block = "uart", instance = 2, io = "tx" }]
+no_default_out = true
+
```

Comment:
I did put them at the end intentionally to avoid altering other mappings. In particular the UART 2 RX input selector. Previously RS485 sat at the end so anything else that wanted to mux a pin to the UART 2 RX input wouldn't now need to change which input it selected.

The real issue here was lack of constants for the actual selectors being generated by the pinmux generator, though maybe we do have that and I haven't noticed!

Certainly this change will need a fix in the rs485 check at least.

Reply:
I appreciate wanting to avoid breaking other mappings, but I believe the RS485 check is the only software that is affected by this change. So I guess it's a question of what's preferred.

What register mapping do you prefer?